tests/scripts/all.sh

```diff
@@ -2159,14 +2161,39 @@ component_release_test_valgrind_constant_flow () {
     # Test asm path in constant time module - by default, it will test the plain C
     # path under Valgrind or Memsan. Running only the constant_time tests is fast (<1s)
     msg "test: valgrind asm constant_time"
-    scripts/config.py --force set MBEDTLS_TEST_CONSTANT_FLOW_ASM
     skip_all_except_given_suite test_suite_constant_time
     cmake -D CMAKE_BUILD_TYPE:String=Release .
     make clean
     make
     make memcheck
 }
 
+component_release_test_valgrind_constant_flow_no_asm () {
```

Comment:
Note: `release` components don't run in the pull request CI. Therefore, when we make a direct change to release components, we need to run the Linux part of the [release job](https://mbedtls.trustedfirmware.org/job/mbedtls-restricted-release-new/) on the PR branch.

Reply:
I might not have access because it says 'page not found'

Reply:
You need to be on the internal network or VPN. If you still can't access it, let me know and I'll start the job.

Reply:
Ah, sorry, commented too fast: the link was to the Open CI so that's not the issue. You need to log in before you can access that job (or manually start a build for any job, for that matter). You should be able to log in with github and your github group membership should give you appropriate permissions on Open CI. If that's not the case, please shout on slack.

Reply:
Forgot you're on holiday this week. I just started the job, then.